<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicInfoItem</class>
 <widget class="QWidget" name="MusicInfoItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="favor_icon_btn">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#favor_icon
{
	background-color: rgb(0, 255, 255);
	background-repeat: no-repeat;
	background-position: center center;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="music_name">
     <property name="text">
      <string>歌名</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="singer">
     <property name="text">
      <string>歌手名称</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="album_label">
     <property name="text">
      <string>专辑名称</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
